sources/api/controllers/GroupController.ts

```diff
@@ -106,3 +106,66 @@ export async function deleteGroup(req: Request, res: Response) {
         message: "Group successfully deleted"
     });
 }
+
+//
+// Kick multiple users from a group
+//
+export async function kickFromGroup(req: Request, res: Response) {
+    const requestingUser = getRequestingUser(req);
+    const groupId = req.params.group_id;
+    const user_list = req.body.users;
+
+    // Check received arguments
+    if (!user_list || !user_list.length)
+        throw new ApiException(400, 'Invalid users argument.');
+
+    const conn = await DbConnection.getConnection();
+    const groupRepository = conn.getRepository(Group);
+    const userRepository = conn.getRepository(User);
+    const group = await Group.getGroupInfo(groupRepository, groupId);
+
+    // Check if group exists
+    if (!group)
+        throw new ApiException(403, 'Invalid group ID');
+
+    // Check if calling user is indeed a member of the group
+    if (group.members.map((e: User) => e.id).indexOf(requestingUser.id) === -1)
+        throw new ApiException(403, 'You are not a member of the group');
+
+    for (let user of user_list) {
+
+        let user_id: number = (await getUser(userRepository, user)).id;
+
+        // Check if user is a member of the group
+        if (group.members.map((e: User) => e.id).indexOf(user_id) === -1)
+            throw new ApiException(404, `User ${user} is not a member of the group.`);
+
+        // Check if calling user has proper rights to remove other users
```

Comment:
```suggestion
        // Check if calling user is removing itself
        // or has proper rights to remove other users
```

I think it's clearer this way (at least this is what I think you are trying to do here)